build.sbt

```diff
@@ -20,22 +20,22 @@ organization := "io.delta"
 
 scalaVersion := "2.12.10"
 
-sparkVersion := "3.0.0"
+val sparkVersion = "3.0.0"
```

Comment:
This is a temporary change. Previously this key was imported by sbt-spark-packages. I had to disable that plugin because it doesn't have a SBT-1.x/Scala 2.12 release.